src/index.html

```diff
@@ -8,25 +8,27 @@
 </head>
 <body>
   <header>
-      <h1>LeaderBoard</h1>
+      <h1 class="neon">LeaderBoard</h1>
     </header>
 
     <section class="score-board">
       <div class="board">
         <div class="board-header">
-          <h2>Recent Scores</h2>
+          <h2 class="glitch" data-text="RECENTSCORES">Recent Scores</h2>
+          <h2 class="glow">Recent Scores</h2>
           <button type="button" class="btn refresh-btn">Refresh</button>
```

Comment:
- [OPtional] Firstly I must commend your design, it looks good, I would suggest that on first load the app shows all the previously entered scores from other users, so the new user sees what's already there. But on your design, this only shows when the user clicks refresh to see his or her score.

God job Happy coding :+1:

Reply:
Hey @Ginohmk, Thanks for the tip. I've implemented it.